editor/editor_properties.cpp

```diff
@@ -2983,8 +2983,17 @@ bool EditorPropertyResource::_is_drop_valid(const Dictionary &p_drag_data) const
 	String allowed_type = base_type;
 
 	Dictionary drag_data = p_drag_data;
-	if (drag_data.has("type") && String(drag_data["type"]) == "resource") {
-		Ref<Resource> res = drag_data["resource"];
+
+	Ref<Resource> res;
+	if (drag_data.has("type") && String(drag_data["type"]) == "script_list_element") {
+		ScriptEditorBase *se = Object::cast_to<ScriptEditorBase>(drag_data["script_list_element"]);
+		res = se->get_edited_resource();
+	}
+	else if (drag_data.has("type") && String(drag_data["type"]) == "resource") {
```

Comment:
Style issue, this should be on the previous line.  See https://docs.godotengine.org/en/latest/community/contributing/code_style_guidelines.html

Reply:
Fixed, sorry! I thought I had my IDE configured to format on save.